packages/css-property-parser/lib/Parser.re

```diff
@@ -1974,6 +1974,7 @@ and symbol = [%value.rec "<string> | <image> | <custom-ident>"]
 and target = [%value.rec
   "<target-counter()> | <target-counters()> | <target-text()>"
 ]
+and url_with_interp = [%value.rec "<url> | url( <interpolation> )"]
```

Comment:
and from the other point of view, when url isn't going to be interpolatable? maybe do a url with interpolation and call it url? wdyt?